CppCoreGuidelines.md

```diff
@@ -1758,22 +1758,17 @@ Make the interface precisely specified and compile-time checkable in the (not so
 Use the C++20 style of requirements specification. For example:
 
     template<typename Iter, typename Val>
-    // requires InputIterator<Iter> && EqualityComparable<ValueType<Iter>, Val>
+      requires input_iterator<Iter> && equality_comparable_with<iter_value_t<Iter>, Val>
     Iter find(Iter first, Iter last, Val v)
     {
         // ...
     }
 
-##### Note
-
-Compilers that support C++20 are able to check `requires` clauses once the `//` is removed.
-Concepts are supported in GCC 6.1 and later.
-
 **See also**: [Generic programming](#SS-GP) and [concepts](#SS-concepts).
 
 ##### Enforcement
 
-(Not yet enforceable) A language facility is under specification. When the language facility is available, warn if any non-variadic template parameter is not constrained by a concept (in its declaration or mentioned in a `requires` clause).
+Warn if any non-variadic template parameter is not constrained by a concept (in its declaration or mentioned in a `requires` clause).
```

Comment:
Not related to this PR, but this needs to suggest a way to suppress the warning. Preferable something simpler than everyone introducing their own form of `template<class> concept Auto = true;`.

Reply:
For example,
```C++
template<std::regular> struct X { };
template<class T>
  std::common_type<X<T>, X<T>> { };
```
would have a warning on line 2.